spk/mosquitto/src/dsm-control.sh

```diff
@@ -7,7 +7,6 @@ DNAME="Mosquitto"
 # Others
 INSTALL_DIR="/usr/local/${PACKAGE}"
 PATH="${INSTALL_DIR}/bin:${PATH}"
-USER="mosquitto"
```

Comment:
why? if you do this you'll need to handle start/stop of SPK too: https://github.com/sarfata/spksrc/blob/7e39296a747d48cd019aa47bbcedcb54578ab291/spk/mosquitto/src/dsm-control.sh#L17